query/src/main/java/org/infinispan/query/Search.java

```diff
@@ -37,61 +37,61 @@ private Search() {
       return makeFilter(queryString, null);
    }
 
    /**
-    * Create an event filter out of an Ickle query string.
+    * Creates event filters from Ickle query strings.
     */
    public static <K, V> CacheEventFilterConverter<K, V, ObjectFilter.FilterResult> makeFilter(String queryString, Map<String, Object> namedParameters) {
       IckleFilterAndConverter<K, V> filterAndConverter = new IckleFilterAndConverter<>(queryString, namedParameters, ObjectReflectionMatcher.class);
       return new IckleCacheEventFilterConverter<>(filterAndConverter);
    }
 
    /**
-    * Create an event filter out of an Ickle query.
+    * Creates event filters from Ickle query strings.
     */
    public static <K, V> CacheEventFilterConverter<K, V, ObjectFilter.FilterResult> makeFilter(Query<?> query) {
       return makeFilter(query.getQueryString(), query.getParameters());
    }
 
    /**
-    * Obtain the query factory for building DSL based Ickle queries.
+    * Obtains a query factory to build DSL-based Ickle queries.
     */
    public static QueryFactory getQueryFactory(Cache<?, ?> cache) {
       if (cache == null) {
-         throw new IllegalArgumentException("cache parameter must not be null");
+         throw new IllegalArgumentException("The cache parameter cannot be null.");
       }
       AdvancedCache<?, ?> advancedCache = cache.getAdvancedCache();
       if (advancedCache == null) {
-         throw new IllegalArgumentException("The given cache must expose an AdvancedCache");
+         throw new IllegalArgumentException("The given cache must expose an AdvancedCache interface.");
       }
       checkBulkReadPermission(advancedCache);
       QueryEngine<?> queryEngine = ComponentRegistryUtils.getEmbeddedQueryEngine(advancedCache);
       return new EmbeddedQueryFactory(queryEngine);
    }
 
    /**
-    * Obtain the {@link ContinuousQuery} object for a cache.
+    * Obtains the {@link ContinuousQuery} object for a cache.
     */
    public static <K, V> ContinuousQuery<K, V> getContinuousQuery(Cache<K, V> cache) {
       return new ContinuousQueryImpl<>(cache);
    }
 
    private static <K, V> AdvancedCache<K, V> getAdvancedCache(Cache<K, V> cache) {
-      AdvancedCache<K, V> advancedCache = Objects.requireNonNull(cache, "cache parameter must not be null").getAdvancedCache();
+      AdvancedCache<K, V> advancedCache = Objects.requireNonNull(cache, "The cache parameter cannot be null.").getAdvancedCache();
       if (advancedCache == null) {
-         throw new IllegalArgumentException("The given cache must expose an AdvancedCache interface");
+         throw new IllegalArgumentException("The given cache must expose an AdvancedCache interface.");
       }
       checkBulkReadPermission(advancedCache);
       return advancedCache;
    }
 
    /**
-    * @return Obtain the {@link Indexer} instance for the cache.
+    * @return Obtains the {@link Indexer} instance for a cache.
     * @since 11.0
```

Comment:
@gustavonalle There are no descriptions for `getSearchStatistics`, `getClusteredSearchStatistics`​, or `getIndexer`​ - Shall I add them or is there a reason why we shouldn't add descriptions?

Reply:
I didn't add them because I thought the name was self-explanatory, feel free to add it